### DISCUSSION TITLE
man

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1709] [General]
imagine posting fake highscores
![Screenshot 2021-08-29 1 56 36 PM](https://user-images.githubusercontent.com/86753001/131262241-cee15433-cc7e-473f-8f69-f9700a202f9f.png)
![Screenshot 2021-08-29 1 56 29 PM](https://user-images.githubusercontent.com/86753001/131262246-448971d4-edb4-4843-aeba-2fcabc07023f.png)
![Screenshot 2021-08-29 1 56 18 PM](https://user-images.githubusercontent.com/86753001/131262247-59ba7f0f-1413-4825-be24-eee03b25f27f.png)

**Comment by Cheemsandfriends** [comment]
Yes.
Imagine telling their friends 'yes, I have 1 billion points on Bopeebo easy'

**Reply by AbnormalPoof** [reply]
LMAO TRUE
